Application/app/src/main/res/values/custom.xml

```diff
@@ -35,7 +35,8 @@ permissions and limitations under the License.
     <!-- Color of splash screen spinning loader animation -->
     <color name="spinner_color">#FFFFFFFF</color>
     <!-- The color of button highlights and widgets. -->
-    <color name="accent">#00AEEF</color>
+<!--<color name="accent">#00AEEF</color>-->
```

Comment:
Could we keep the accent color?
It was set to be different from default Fire App Builder accent color for easy testing.

Reply:
reverted the color.

@ech89899 i cannot use the EPG as library as i need to make custom changes in the library to align the UI as per specs provided, so have to use the source code.